src/huggingface_hub/__init__.py

```diff
@@ -30,6 +30,11 @@
     TF2_WEIGHTS_NAME,
     TF_WEIGHTS_NAME,
 )
+from .fastai_utils import (
+    from_pretrained_fastai,
+    push_to_hub_fastai,
+    save_fastai_learner,
+)
```

Comment:
WDYT of moving integrations out of the root namespace? Having them all in the root namespace makes `import huggingface_hub` pretty heavy.

I would suggest something like:

```
huggingface_hub.core
huggingface_hub.fastai
...
```